core/src/server/rpc_module.rs

```diff
@@ -820,6 +960,34 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 		Ok(callback)
 	}
 
+	/// As [`Self::register_subscription`] but replaces and returns the method
+	/// if it already exists.
+	pub fn replace_subscription<R, F, Fut>(
+		&mut self,
+		subscribe_method_name: &'static str,
+		notif_method_name: &'static str,
+		unsubscribe_method_name: &'static str,
+		callback: F,
+	) -> Option<MethodCallback>
+	where
+		Context: Send + Sync + 'static,
+		F: (Fn(Params<'static>, PendingSubscriptionSink, Arc<Context>, Extensions) -> Fut)
+			+ Send
+			+ Sync
+			+ Clone
+			+ 'static,
+		Fut: Future<Output = R> + Send + 'static,
+		R: IntoSubscriptionCloseResponse + Send,
+	{
+		let prev = self.methods.remove(subscribe_method_name);
+		self.methods.remove(unsubscribe_method_name);
```

Comment:
You can add a comment that this callback is generated jsonrpsee and that's is why it's just removed and not returned here